### ISSUE TITLE
o is undefined

### ISSUE THREAD

**Issue by Geronium** [dbeaver/cloudbeaver#942]
**Describe the bug**
when we open a table with a double click we have this error "o is undefined"

**To Reproduce**
1. Go to a schema
2. click right on a "tables"
3. select one table and double dick on this table to open it
4. See error : "o is undefined" appair inside a popup at the left of the navigator window

**Screenshots**

![Capture d’écran de 2022-06-30 09-51-44](https://user-images.githubusercontent.com/73563389/176624437-134d3d37-0b5e-43b6-8f08-95698dc52bce.png)

**Desktop (please complete the following information):**
 - OS: ubuntu 16.0.4
 - Browser firefox
 - Version 91.4.1esr (64 bits)

**Additional context**
Add any other context about the problem here, e.g. error log.

thank you

**Comment by kseniiaguzeeva** [comment]
Could you please also tell your database?
Can the table be opened via it's context menu?
What Cloudbeaver version do you use?

**Comment by Geronium** [comment]
Hello,

i have this error with the two databases : oracle and postgres. The tables open in any case, I just have this error which opens a popup (apparently without consequences)

here the version

Build time
June 27, 2022
Backend version
22.1.1.202206271843
Frontend version
22.1.1.202206271847

Thank you

**Comment by Wroud** [comment]
Could you please try to press Shift+F5 on the application tab (it's should clean browser cache for cloudbeaver)

**Comment by kseniiaguzeeva** [comment]
I've reproduced the issue on Chrome version 81.0.4044.129. Could you please try to update the browser version?

**Comment by kseniiaguzeeva** [comment]
I have noticed that you have not updated your ticket for a long time. If you wish to reopen this ticket, please feel free to contact me.

**Comment by UnleashSpirit** [comment]
Hello,
I have the same problem, it occurs every time I resize the view (left panel, show/hise VALUE panel, etc)
```javascript
const u = new o((e=>{
  const o = e[0].contentBoxSize[0];
  r($(o.inlineSize)), // <== Here
  i(o.blockSize)
}));
```
I'm using dbeaver/cloudbeaver:latest (2 days ago) and it happens only on Firefox (Chrome & Edge are OK)

**Comment by kseniiaguzeeva** [comment]
It's a browser related problem. Could you please use another browser version (e.g. 103.0.2 or MSIX -1.0)?

**Comment by kseniaguzeeva** [comment]
The bug is fixed and available in the latest 22.2.0 release.
